### PR TITLE
pulled repositories on github for projects

### PR DESCRIPTION
**Pull request by kjpardillo**
pulled repositories on github for projects

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -281,7 +281,10 @@ footer {
     display: block;
     transition: 0.3s;
     }
-
+.project_links{
+    color: inherit;
+    text-decoration: none;
+}
```
Comment (akosasante) [inline css/index.css:287]: Just curious to understand why you decided to style the links this way? In my opinion, it makes it a bit difficult to see/realize initially that these items are links on the page apart from the cursor changing when I hover over them. Did you feel it fit more into the style you were going for?

**Review by akosasante — COMMENTED** [review]
Great job here Ki!   
 

Please let me know if you had any trouble with any of the concepts or any questions about AJAX/making async calls from the client to the server.

**Comment by kjpardillo** [comment]
Hi,

I am actually going to change that design for my aubmiasion to my final project
________________________________

File: js/index.js [hunk 2]
```diff
@@ -79,3 +79,23 @@ document.getElementById("mySidebar").style.width = "0";
...
+  console.log(repositories);
+  const projectSection = document.getElementById('projects');
+  const projectList = projectSection.querySelector('ul');
+
+  for (let i = 0; i < repositories.length; i +=1) {
+    let project = document.createElement('li');
+    project.innerHTML = `<a class= "project_links" href="${repositories[i].html_url}">${repositories[i].name}</a>`;
+    
+    projectList.appendChild(project);
+  };
```
Comment (akosasante) [inline js/index.js:99]: no semicolon needed after `for` loop braces

**Comment by akosasante** [comment]
> Hi, I am actually going to change that design for my aubmiasion to my final project

Okay, sounds good 👍  Looking forward to seeing the final design in action!

**Pull request merged**